### PR TITLE
[IMP] calendar: convert qunit tests to hoot

### PR DESCRIPTION
**Pull request by rapr-odoo**
This PR aims to convert remaining QUnit tests to hoot.

Part of task-3818666

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/calendar/__manifest__.py [hunk 1]
```diff
@@ -48,14 +48,11 @@
         # Unit test files
         'web.assets_unit_tests': [
             'calendar/static/tests/**/*.js',
-            ('remove', 'calendar/static/tests/legacy/**/*'),  # to remove when all legacy tests are ported
             ('remove', 'calendar/static/tests/helpers/**/*'),
             ('remove', 'calendar/static/tests/tours/**/*'),
         ],
         'web.qunit_suite_tests': [
-            'calendar/static/tests/legacy/**/*',
             'calendar/static/tests/helpers/**/*',
```
Comment (tsm-odoo) [inline addons/calendar/__manifest__.py:55]: Are you sure helpers are still needed in qunit?

**Review by tsm-odoo — APPROVED** [review]
@robodoo delegate+

LGTM otherwise.

**Comment by tsm-odoo** [comment]
@robodoo r-
@robodoo delegate+

File: addons/calendar/__manifest__.py (hunk 1 above)
Comment (rapr-odoo) [reply addons/calendar/__manifest__.py:55]: Yes. `google_calendar` and `microsoft_calendar` still needs it.

**Comment by tsm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**